model/src/main/proto/interaction_object.proto

```diff
@@ -30,6 +31,11 @@ message StringList {
   repeated string html = 1;
 }
 
+// Structure containing a ratio object.
+message RatioExpression {
+  string html = 1;
```

Comment:
Why wouldn’t we use a structured representation here like we would in the backend? We shouldn’t just store html that needs parsing. The data in there protos should be already structured (eg we should be able to easily observe the number of ratios, the components of the ratio, etc).

Reply:
I'm not sure if I'm following this correctly but does this mean we don't want to create a object of ratio explicitly in Android and can use the existing one's ?

Reply:
@BenHenning updated